.buildkite/lm-eval-harness/configs/Meta-Llama-3-8B-Instruct-Channelwise-compressed-tensors.yaml

```diff
@@ -4,8 +4,8 @@ tasks:
 - name: "gsm8k"
```

Comment:
nit: it would be nice to rename this config to `Meta-Llama-3-8B-Instruct-W8A8-FP8-Channelwise-compressed-tensors.yaml`